src/Components/Navbar/Components/NavbarMenu/NavbarMenu.scss

```
@@ -0,0 +1,109 @@
+.NavbarMenu {
+  width: 100%;
+
+  ul.gnbMenu {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    li {
+      margin: 15px;
+
+      a {
+        display: block;
+        font-size: 14px;
+        color: rgb(50, 50, 50);
+      }
+    }
+
+    li.myPolo a {
+      color: #dd1785;
+    }
+  }
+  .NavbarSubMenu {
+    position: absolute;
+    left: 0;
+    width: 100%;
+    height: 0;
+    overflow: hidden;
+    transition: height 0.5s ease;
+    background-color: #fff;
+
+    .subList {
+      display: flex;
+      padding: 25px 60px;
+      align-items: flex-start;
+      justify-content: space-between;
+      animation: fadeOut 1s ease-in-out;
+
+      .list {
+        animation: fadeOut 1s;
+
+        h2 {
+          margin-bottom: 10px;
+          font-size: 12px;
+          color: rgb(72, 72, 72);
+        }
+
+        ul {
+          line-height: 1.8;
+
+          li {
+            a {
+              font-size: 13px;
+              color: rgb(38, 38, 38);
```

```suggestion
            a {
              font-size: 13px;
              color: rgb(38, 38, 38);
```
필요한 만큼만 써주시면 될거 같습니다.